### PR TITLE
Added original templates

### PR DESCRIPTION
**Pull request by alejoe91**

### CODE REVIEW

File: MEArec/tools.py [hunk 1]
```diff
@@ -327,6 +327,10 @@ def load_recordings(recordings, return_h5_objects=True,
                 rec_dict['templates'] = F.get('templates')
             else:
                 rec_dict['templates'] = np.array(F.get('templates'))
+            if return_h5_objects:
+                rec_dict['original_templates'] = F.get('original_templates')
```
Comment (alejoe91) [inline MEArec/tools.py:331]: I think here we should check if it exists for backcompatibility

Comment (samuelgarcia) [reply MEArec/tools.py:331]: ok

**Comment by samuelgarcia** [comment]
I push change for backwards compatibility.
Can you check ? (and merge ?)

**Comment by alejoe91** [comment]
Looks good Sam! Waiting for the tests and merging

**Pull request merged**